src/components/AirdropWalletsCarouselItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import { CONST, AirdropWalletDetails, AirdropGoal } from 'app/consts';
+
+import { AirdropWalletBalance } from './AirdropWalletBalance';
+import { AirdropWalletFooter } from './AirdropWalletFooter';
+
+const i18n = require('../../loc');
+
+const airdropGoals: AirdropGoal[] = [
```

Comment:
I'm not a fan when a function depends on external variables. I'd rather see it inside `AirdropWalletsCarouselItem` or it should be injected in props.

Reply:
my logic always is that static variables should go outside of the function as the function does not fiddle with them anyhow, so keeping them inside is more of a polution. Also whole body of functional component is treated as render, so whenever component rerender, you do all those assignments again and again. Rather minor, but still counts :P 
Also I believe that I have already seen "consts out" strategy somewhere in the project. 
Let s have @KedziaPawel as a judge, as after his decision it will be 2:1 for one or the other :)

Reply:
Now when I think about it can you give some arguments for what you prefer? That may convince me or quite the oposite :D

Reply:
normally I would keep it as it is but here it cannot be done like this because when you will change the language in the app you will get old language translations

Reply:
hmm, true, so we have to keep it inside, so it rerenders properly without force refreshing whole app, right?. Good point, will move it.

Reply:
@jellyfish-tom I simply believe that we should at least try to develop our components according to the "same input same output" rule. I don't find it a good practice to have a component or function that depends on some external states. It's like using brackets in `if else` - in general, it's fine until someone doesn't make a mistake :)

Also, these two variables, are strongly bonded with the component that's why I believe they should be a part of it

Reply:
functional React components are functions but I think logic you apply here and the rules you mention (with which I agree) are not applicable in such a straight manner. Reason being rerenders that I mentioned before. Also there s kinda no place for mistake you mention. 
Code I depend on is not a function, so input is always the same, cause logically whole file is treated as an 'input' in this case. It s not possible to change it without actually reimplementing it and change by reimplementation is what you can do with whichever file you pick, say `package.json`.
Only way to change it "from the outside" without being aware is to change translations. If I do what you say and move it inside and then change something in translations result is the same, which is yet another proof that "input" is in thise case whole file, not function body. You actually write that yourself "by being part of the component" - they are part of the component as whole file is a component, not only function body. Hope I convinced you :D

Reply:
let's say you did 🙃 it actually makes sense. Thanks